examples/simple_cache/main.cpp

```diff
@@ -22,7 +22,7 @@ namespace cachelib_examples {
 using Cache = cachelib::LruAllocator; // or Lru2QAllocator, or TinyLFUAllocator
 using CacheConfig = typename Cache::Config;
 using CacheKey = typename Cache::Key;
-using CacheItemHandle = typename Cache::ItemHandle;
+using CacheItemHandle = typename Cache::ReadHandle;
```

Comment:
Can you rename this to `CacheReadHandle`? Also need to update L47 where CacheItemHandle is used. (We have since split ItemHandle into Read/Write handle to enforce r/w semantics).

Reply:
Thanks, I have made the changes, can you reimport the pull request?